js/templates/userShort.html

```diff
@@ -1,29 +1,31 @@
 <div class="contentBox clrBr clrP clrSh2">
   <div class="hero bkgCover"
-    <% if(ob.header){ %>
-      style="background-image: url('<%= ob.header %>, url('../imgs/defaultHeader.png')"
+    <% if(ob.headerHash){ %>
+      style="background-image: url(<%= ob.getServerUrl(`ipfs/${ob.headerHash}`) %>), url('../imgs/defaultHeader.png')"
     <% } else { %>
       style="background-image: url('../imgs/defaultHeader.png')"
     <% } %>>
     <div class="userIcon disc clrBr2 clrSh1"
-      <% if(ob.avatar){ %>
-       style="background-image: url('<%= ob.avatar %>, url('../imgs/defaultUser.png')"
+      <% if(ob.avatarHash){ %>
+        style="background-image: url(<%= ob.getServerUrl(`ipfs/${ob.avatarHash}`) %>), url('../imgs/defaultBanner.png')"
       <% } else { %>
        style="background-image: url('../imgs/defaultUser.png')"
       <% } %>>
     </div>
-    <div class="userControls">
-      <% if(!ob.ownGuid) { %>
-        <a class="iconBtn clrP clrBr js-follow <% if(ob.followedByYou){ %>active<% } %>">
-          <i class="ion-person-stalker"></i>
-        </a>
-        <% if(ob.moderator) { %>
-          <a class="iconBtn clrP clrBr js-mod <% if(ob.ownMod){ %>active<% } %>">
-            <i class="ion-briefcase"></i>
+    <% if(!ob.hideControls){ %>
+      <div class="userControls">
+        <% if(!ob.ownGuid) { %>
+          <a class="iconBtn clrP clrBr js-follow <% if(ob.followedByYou){ %>active<% } %>">
+            <i class="ion-person-stalker"></i>
           </a>
+          <% if(ob.moderator) { %>
+            <a class="iconBtn clrP clrBr js-mod <% if(ob.ownMod){ %>active<% } %>">
```

Comment:
lol, i got to find a linter for our templates so we can have some spaces after an 'if' and before the curly brace!